services/translateService.js

```diff
@@ -1,27 +1,8 @@
 const {
-  translate: googleTranslate,
-} = require("@vitalets/google-translate-api");
-const { translate: bingTranslate } = require("./bingTranslate");
-const { translate: libreTranslate } = require("./libreTranslate");
-const {
-  getTranslationEnginesToUse,
-} = require("../utils/getTranslationEnginePreferences");
-const { validateLanguageIsSupportedByEngine } = require("../utils/validateLanguageIsSupportedByEngine");
-
-const translateEngines = {
-  google: googleTranslate,
-  bing: bingTranslate,
-  libreTranslate,
-};
-
-const validEngines = Object.keys(translateEngines);
-
-/**
- * Checks if the given translation engine is valid.
- * @param {string} engine - The translation engine to check.
- * @returns {boolean} - True if the given translation engine is valid, false otherwise.
- */
-const isEngineValid = (engine) => validEngines.includes(engine);
+  getLanguageCodeByEngine,
+} = require("../utils/supportedLanguagesUtils");
+const { getTranslationEnginesToUse } = require("../utils/getTranslationEnginePreferences");
+const { validEngines, translateEngines, isEngineValid } = require("../utils/translationEnginesUtils");
```

Comment:
Moved to another file to avoid circular dependencies with supportedLanguagesUtils file https://www.linkedin.com/pulse/circular-dependency-nodejs-modules-nihal-kaul/